api/glossary_usefulconcepts.md

```diff
@@ -0,0 +1,156 @@
+Glossary of useful concepts
+============================
+###### Narrowing the search space when you don’t even know what you don’t know
+
+## Backend technology stack
+This is divided into two sections. The technology stack is described first, as an introduction to the backend technologies used in Baobab. All words in **bold** are also defined within this document.
+
+### Python
+[Python](https://www.python.org/doc/essays/blurb/) is an interpreted object-orientated, high level programming language with dynamic semantics.
+Let’s break that up:
+
+**1. Interpreted versus compiled languages:**
+
+Languages can have implementations of both
+
+- Compiled:
+written code is converted directly into machine code that the processor executes
+More control over hardware (memory, CPU usage). Examples are C, C++, Go
+- Interpreted: 
+these languages will run through the programme line by line and execute each.
+Examples include JavaScript, Python
+
+**2. Object-orientated programming**
+- This centres around the creation of objects that are characterised by attributes and behaviour. 
+- **Classes** are used in OOP in the creation of objects. High-level versus low-level languages
```

Comment:
Was "High level versus [...]" meant to be another heading (or part of the next bullet point)?